docs/tools.md

```diff
@@ -0,0 +1,190 @@
+# The Lean tool chain
+
+## Introduction
+
+In this document we explain how all the various pieces of the Lean tools ecosystem fit together. 
+Understanding this precisely is not required, but it will help if you decide to explore non-recommended setups.
+In particular we will gradually introduce new pieces in the story, starting with a bare bone Lean. 
+This is *not* the recommended setup.
+Also this document is long, that's the price for not wanting to blindly follow the recommended setup instructions.
+
+## Installing Lean and checking files
+
+### Installing Lean
+
+Of course the main piece is Lean itself. You can download Lean binaries for your OS [here](https://github.com/leanprover-community/lean/releases). 
+This contains folders `bin`, `include` and `lib`.
+The `bin` folder contains a `lean` executable which you can run.
+
+That's all, Lean is installed!
```

Comment:
```suggestion
That's the bare minimum necessary to run Lean files!
```

This document explains two thing: 1) how the tooling fits together under the hood, 2) and how to make a non-recommended setup.

This section is the only section that causes the reader to end up with a non-recommended setup.  The other sections are completely safe and do not require a non-recommended setup at all.  I think it should come with a bigger warning.

I expect that a lot of people will read this text to learn more about the internals.  Those people should still use `elan`, `leanproject`, etc.